docs/user/links-preview.rst

```diff
@@ -0,0 +1,25 @@
+Links preview
+=============
+
+Links previews allows you to see the content of the link you are about to click when you hover it.
+This content will be rendered inside a popup that disappears after moving the mouse outside of it.
```

Comment:
```suggestion
Link previews allows you to see the content of the link you are about to click when you hover over it.
This gives you the ability to quickly preview where a link points,
so you can keep your context while reading but learn more from other resources. 
```

I think it's useful to give the "why should I care?" to the user in the first couple sentences.